internal/pkg/manifest/interpolate.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package manifest
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+)
+
+const (
+	predefinedEnvVarKeyForAppName = "COPILOT_APPLICATION_NAME"
+	predefinedEnvVarKeyForEnvName = "COPILOT_ENVIRONMENT_NAME"
+)
+
+var (
+	// Modified from docker/compose
+	interpolatorRegexp = regexp.MustCompile(`\${([_a-zA-Z][_a-zA-Z0-9]*)}`)
```

Comment:
What do you think of something like
```suggestion
	interpolatorEnvVarRegExp = regexp.MustCompile(`\${([_a-zA-Z][_a-zA-Z0-9]*)}`)
```